ci-operator/step-registry/sandboxed-containers-operator/create-prowjob/sandboxed-containers-operator-create-prowjob-commands.sh

```diff
@@ -246,9 +246,8 @@ command_create() {
     # Generate the prowjob configuration file
 
     cat > "${OUTPUT_FILE}" <<EOF
-documentation: |-
-  DO NOT EDIT DIRECTLY.
-  This is generated by the sandboxed-containers-operator-create-prowjob-commands.sh script.
+#  DO NOT EDIT DIRECTLY.
+#  This is generated by the sandboxed-containers-operator-create-prowjob-commands.sh script.
```

Comment:
Good catch, documentation is only supported for steps/workflows and such, not the generated jobs.